### ISSUE TITLE
Flux Queries With V1 Client Against OSS 2.x

### ISSUE THREAD

**Issue by kbredemeier** [mneudert/instream#78]
The `org` is not added to the URL parameters when using a connection configured for v1 against an InfluxDB OSS 2.x.
The [API documentation](https://docs.influxdata.com/influxdb/v2.4/api/#operation/PostQuery) states that either one of those is required in the query parameters.

```elixir
defmodule V1 do
  use Instream.Connection,
    config: [
      auth: [
        method: :token,
        token: System.fetch_env!("LB_INFLUXDB_LOCAL_TOKEN")
      ],
      org: "instream_test",
      version: :v1,
      database: "mapped_database"
    ]
end

V1.query(
  """
  import "influxdata/influxdb/schema"

  schema.measurements(bucket: "test_bucket")
  """,
  query_language: :flux
)

%{code: "invalid", message: "failed to decode request body: Please provide either orgID or org"}
```

I'm currently working on a fix.

**Comment by mneudert** [comment]
Well, the `:v1` client is was never intended to work with an InfluxDB v2.x instance. And I would argue it never should.

As you are using a Flux query with an Org and querying a Bucket, what is missing to configure the connection as `version: :v2`?

**Comment by mneudert** [comment]
As the current behaviour is not really a bug but it would also be quite painful to having the requirement of adding an option to however many function calls there are I would like to add another option: !xx

This should allow you to keep your code as is and only require `version: :v2, query_language: :influxql` in your config. Unless I messed up somewhere of course.

**Comment by kbredemeier** [comment]
Hey, thanks for the effort. I just gave your branch a spin but I ran into another problem with that: We are still relying on the DBRP mapping for backwards compatibility. If I configure the client with `v2` it doesn't use the [compatibility API](https://docs.influxdata.com/influxdb/cloud/api/#operation/PostLegacyWrite) anymore which breaks our data ingestion.

We could add another option for that but I'm not sure if I'm a fan of that idea :sweat_smile:
